tests/Makefile.am

```diff
@@ -94,18 +92,21 @@ YAMI_VPP_CFLAGS = \
 	$(NULL)
 
 yamidecode_LDADD    = $(YAMI_VPP_LIBS)
-yamidecode_LDFLAGS  = $(YAMI_VPP_LDFLAGS)
+yamidecode_CPPFLAGS = $(YAMI_COMMON_CFLAGS)
+yamidecode_LDFLAGS  = $(YAMI_VPP_LDFLAGS) $(AM_LDFLAGS)
```

Comment:
`$(YAMI_VPP_LDFLAGS)` doesn't appear to be defined anywhere.  Thus, we can remove this override.